polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -402,6 +402,17 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> EventEmitter for Pallet<T> {
+		fn emit_sent_event(
+		origin: Location,
+		destination: Location,
+		message: Xcm<()>,
+		message_id: XcmHash,
+		) {
+			Self::deposit_event(Event::Sent { origin, destination, message, message_id });
```

Comment:
would not reuse `Event::Sent` which is used by `pallet_xcm::send()`

```suggestion
			Self::deposit_event(Event::XcmExecutorSent { origin, destination, message_id });
```

Reply:
Why would you not reuse it? Are they really that different?
If they're just one, apps can just subscribe to one event and know all cross-chain messages from that.

Reply:
true, on second thought we could reuse, I am still not sure about including the full message in the event though, some messages could be quite big

Reply:
@jpserrat please print all events generated by some cross-chain emulated test and let's see how much of an effect these changes make on the overall events footprint.

I suggest a bridge test that involves multiple chains: like [this one](https://github.com/paritytech/polkadot-sdk/blob/5f6c8e8fd20f19de0b592ef2413e934e97c9bc68/cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/asset_transfers.rs#L128) or [this one](https://github.com/paritytech/polkadot-sdk/blob/5f6c8e8fd20f19de0b592ef2413e934e97c9bc68/cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/asset_transfers.rs#L542)

Reply:
Assert logs for these two tests